### ISSUE TITLE
Ghost icons on maps

### ISSUE THREAD

**Issue by filipefurtad0** [openfoodfoundation/openfoodnetwork#5633]
## Description

The icons of some existing enterprises are missing and don't appear on the map. So far, only was able to find two cases, each of them in staging-UK and -AU.

Newly created enterprises are displayed correctly.

## Expected Behavior

Active and visible enterprises should be seen on the map with the appropriate icon.

## Actual Behaviour

Some are not.

## Steps to Reproduce

1. Go to staging-uk or staging-au. 
2. Enable dev-tools (F12) and press Ctrl+Shift+C to enable the element picker.
3. Hover on the areas in the pics below, for the respective staging servers.
4. You should be able to find open enterprises, despite the missing icon.

## Animated Gif/Screenshot

Staging-UK (in Firefox 75): Hub The Hacienda
![image.png](https://images.zenhubusercontent.com/5d761b049dde650001eaf32f/bc169f76-2440-495a-8b35-4cb71643abbf)

Staging-AU (in Chrome): Hubs Nicoles Market / Avocado Producer / Carrot Producer

![image.png](https://images.zenhubusercontent.com/5d761b049dde650001eaf32f/7e5e337c-9ae9-489f-a543-28627c3dcca8)![image.png](https://images.zenhubusercontent.com/5d761b049dde650001eaf32f/786fa7c7-4177-42c1-bbd4-ca3bfeef9026)

## Workaround

None.

## Severity

bug-s2: a non-critical feature is broken, no workaround

## Your Environment

* Version used: v3.0 stable
* Browser name and version: Firefox 75 and Chrome 83.0.4103.97 (Official Build) (64-bit)
* Operating System and version (desktop or mobile): Desktop and Mobile

## Possible Fix

Seems just the pic is not visible, everything else is there. Also, it seems to affect only UK and AU (it's not possible to check in FR though) - maybe related to #5622?

**Comment by luisramos0** [comment]
:+1: I am not sure this is an s2. I am not sure this should block the v3 release.

**Comment by filipefurtad0** [comment]
Thank you for the feedback.
I'm unsure on the extension/impact - hence the S2. But I agree not to block the release for this one @luisramos0. Should we downgrade it?

**Comment by Matt-Yorkley** [comment]
I'm assuming this is related to the new icon we added recently that shows "+1" where there are locations that overlap. I think this is related to #5622. Lets take a look again after #5624 is merged.

**Comment by sauloperez** [comment]
Yes, it really looks like an asset precompilation issue.

**Comment by Matt-Yorkley** [comment]
Thanks for spotting it @filipefurtad0 :muscle:

I think there might be some other bits like this with images and icons, we'll need to keep an eye out... :eyes:

**Comment by filipefurtad0** [comment]
Thanks @Matt-Yorkley :-)

I investigated a bit more and it seems indeed related to the +1 sign.

I've set these two hubs real close, in Sheffield - nrs. 149 and 151 Arundel St.:

![image](https://user-images.githubusercontent.com/49817236/85006626-44ec6880-b152-11ea-9e96-c84992ff171b.png)

It's possible to make these icons disappear if the address is set to be exactly the same: it should show the +1 icon.

Downgrading to S2, as this is not a critical feature.
